mininet/node.py

```diff
@@ -838,6 +838,38 @@ def init( cls ):
         mountCgroups()
         cls.inited = True
 
+class XIAHost( Host ):
+
+    "XIA host"
+
+    def __init__( self, name, **kwargs ):
+        ""
+       	Node.__init__( self, name, **kwargs )
+	self.xids = {}        
+
+    def config( self, hid=None, **params ):
+	""
+	r = Node.config( self, **params )
+	self.setParam( r, 'setHID', hid=hid )
+	return r
+		
+    def setHID( self, hid=None):
+	""
+	hidList = self.addParameters( hid )	
+	moduleDeps( add='xia_ppal_hid' )
+	for p in hidList:
+	    self.cmd('xip hid new',p)
+	    self.cmd('xip hid getpub',p)
+	    self.cmd('xip hid addaddr',p)
+
+    def addParameters( self, xid ):
+	""	
+	self.xids['hid'] = []
+	hid_list = xid.split(',')
+	for hid in hid_list:
+	    self.xids['hid'].append(hid)
+	return self.xids['hid']
```

Comment:
Is this method supposed to be generic and work for any `xid`? Or does it only work for HIDs?

If it's only for HIDs, you should change the name of the parameter from `xid` to `hid`. If it's for any XID, then it shouldn't directly use the string `'hid'`.